addons/sales_team/sales_team.py

```diff
@@ -63,6 +63,28 @@ def __get_bar_values(self, cr, uid, obj, domain, read_fields, value_field, group
             team_result[self._period_number - (month_delta.months + 1)] = {'value': group.get(value_field, 0), 'tooltip': group.get(groupby_field, 0)}
         return team_result
 
+    def _get_default_team_id(self, cr, uid, user_id=None, context=None):
+        team_id = self._resolve_team_id_from_context(cr, uid, context=context) or False
+        if not team_id:
+            user = self.pool['res.users'].browse(cr, uid, user_id or uid, context=context)
+            team_id = user.partner_id.team_id.id
```

Comment:
I think this is wrong it would assign the salesperson of the sale, not the first team he is member of.

rest of the commit seems ok.